File: com.vorwardit.wordsworthcms/Models/Page.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;
using System.Web.Script.Serialization;

namespace com.vorwardit.wordsworthcms.Models
{
    public class Page
    {
        public Guid PageId { get; set; }

        public Guid PageLayoutId { get; set; }
        [ForeignKey("PageLayoutId")]
        public PageLayout PageLayout { get; set; }

        [MaxLength(50)]
        public string Name { get; set; }

        public virtual ICollection<PageVersion> Versions { get; set; }

        public virtual ICollection<PageUrl> Urls { get; set; }
    }
}

File: com.vorwardit.wordsworthcms/lib/admin/Domain/Page.js
var app;
(function (app) {
    var domain;
    (function (domain) {
        var Page = (function () {
            function Page(pageId, pageLayoutId, name, urls) {
                this.pageId = pageId;
                this.pageLayoutId = pageLayoutId;
                this.name = name;
                this.urls = urls;
            }
            return Page;
        })();
        domain.Page = Page;
    })(domain = app.domain || (app.domain = {}));
})(app || (app = {}));
